common/app/views/support/AmpAd.scala

```diff
@@ -33,16 +35,40 @@ case class AmpAdDataSlot(article: Article) {
   }
 }
 
-case class AmpAdRtcConfig() {
-  def toJsonString: String =
-    Json.obj("urls" -> (
-               Json.arr() ++
-                 (
-                   if(KruxSwitch.isSwitchedOn)
-                     // See https://trello.com/c/zBlwvOWI
-                     Json.arr("https://cdn.krxd.net/userdata/v2/amp/2196ddf0-947c-45ec-9b0d-0a82fb280cb8?segments_key=x&kuid_key=kuid")
-                   else Json.arr()
-                 )
-             )
-    ).toString()
+object AmpAdRtcConfig {
+
+  def toJsonString: String = {
+
+    val urls: Seq[(String, JsValueWrapper)] = {
+
+      def urlValue(url: String, switch: Switch): Option[String] =
+        if (switch.isSwitchedOn) Some(url)
+        else None
+
+      // See https://trello.com/c/zBlwvOWI
+      val kruxUrl = urlValue(
+        "https://cdn.krxd.net/userdata/v2/amp/2196ddf0-947c-45ec-9b0d-0a82fb280cb8?segments_key=x&kuid_key=kuid",
+        KruxSwitch
+      )
+
+      val prebidServerUrl = urlValue(
+        s"$prebidServerHost/openrtb2/amp?tag_id=1&w=ATTR(width)&h=ATTR(height)&ow=ATTR(data-override-width)" +
+          "&oh=ATTR(data-override-height)&slot=ATTR(data-slot)&targeting=TGT&curl=CANONICAL_URL&timeout=TIMEOUT" +
+          "&adcid=ADCID&purl=HREF",
```

Comment:
Maybe point to the doc?

Reply:
Good idea.  The doc is difficult to find!